logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/scenario/OnHttpRequest.kt

```diff
@@ -56,6 +57,57 @@ class OnHttpRequest(
         }
     }
 
+    fun sendClientDiscoveryMessage(
+        conversationId: ConversationId,
+        userId: UserId,
+        clientId: ClientId,
+    ) {
+        /*
+
+          val message = QualifiedOtrMessage(selfClientId, QEncryptedContent.Empty, nativePush = false)
+            msgClient.postMessage(convId, message).future.map {
+              case Left(error) => Left(error)
+              case Right(resp) => Right(resp.missing)
+            }
+
+         */
+        callingScope.launch {
+            val message = Message(
+                uuid4().toString(),
+                MessageContent.Empty,
+                conversationId,
+                Clock.System.now().toString(),
+                userId,
+                clientId,
+                Message.Status.SENT,
+                Message.EditStatus.NotEdited
+            )
+            when (val sentMessage = messageSender.sendClientDiscoveryMessage(message = message)) {
+                is Either.Right -> {
+                    callingLogger.i("OnHttpRequest() -> sendClientDiscoveryMessage() - Success: ${sentMessage.value}")
+                    callingLogger.i(sentMessage.value)
+
+                    /*
+
+                    val clients = userClients.entries.flatMap { case (userId, clientIds) =>
+                      clientIds.map { clientId =>
+                        AvsClient(userId.str, clientId.str)
+                      }
+                    }
+
+                    val json = encode(AvsClientList(clients.toSeq))
+                    withAvs(wcall_set_clients_for_conv(wCall, convId.str, json))
+
+
+                     */
```

Comment:
I think we should remove this commented code, before merge and/or replace with a todo comment in case is relevant 😅

Reply:
omg, yes!